.gitignore

```diff
@@ -49,3 +49,6 @@ certum.jks
 
 # Ignore icons that exists twice (for reference inside the jar and for the webstarter)
 /src/main/resources/org/terasology/launcher/icons/
+
+# Ignore Web API sources generated by Swagger plugin
+web-api-client/
```

Comment:
Can you choose where the sources are generated to? I'm not sure about the (quite generic) name here.

Reply:
Yep, the directory can be set in build.gradle, using the following: 
`generateSwaggerCodeClient.outputDir = file('web-api-client')`